armi/physics/neutronics/crossSectionGroupManager.py

```diff
@@ -1299,3 +1329,115 @@ def blockCollectionFactory(xsSettings, allNuclidesInProblem):
     return BLOCK_COLLECTIONS[blockRepresentation](
         allNuclidesInProblem, validBlockTypes=validBlockTypes
     )
+
+
+class LatticePhysicsUpdateFrequency(IntEnum):
+    """
+    Enumeration for lattice physics update frequency options.
+    """
+
+    BEGINNING_OF_LIFE = 1
+    BEGINNING_OF_CYCLE = 2
+    EVERY_NODE = 3
+    FIRST_COUPLED = 4
+    ALL = 5
```

Comment:
I know its a bit unorthodox to have class variables that are not all caps, but in this case I'd like to propose an exception. 

If you were to instead change these enum attributes to the names that they correspond to (i.e. make them like `BOL = 1`, `everyNode=4`, etc), then you could avoid having your `_stringMapping()` method and also remove your `getStr()` method in favor of just calling `LatticePhysicsUpdateFrequency.everyNode.name` when you want the string. This seems more in line with the enum philosophy and makes more use of the infrastructure that is already in place.

Not really important, but just a thought.

Reply:
Great idea, this makes the class much simpler.